lib/litmus.ex

```diff
@@ -1,10 +1,21 @@
 defmodule Litmus do
-  @moduledoc File.read!("#{__DIR__}/../README.md")
+  @moduledoc """
+  Litmus is a parameter validation library for Elixir.
```

Comment:
Should we make this `data validation` like we have in the `README.md` above? Or should we change that to be `Parameter validation in Elixir`?